src/components/field/MessageOfDay.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import i18n from '@dhis2/d2-i18n'
+import PropTypes from '@dhis2/prop-types'
+import { TextAreaField } from './TextAreaField'
+
+const CODE = 'messageOfTheDay'
+
+export const MessageOfDay = ({ value, onChange, disabled }) => {
+    const handleChange = e => {
+        onChange({ ...value, [CODE]: e.value })
+    }
+
+    return (
+        <TextAreaField
+            name={CODE}
```

Comment:
Hey,

I am able to write and save texts with more than 200 characters. The Jira ticket mentions a max limit of 200 characters. Can this restriction be passed as a `prop` to `TextArea`? 

Thank you!

Reply:
Hey, I tried using a prop like maxLength, but it didn't work, I'll add a function to limit to 200 characters. Thanks